packages/assets-controllers/src/TokensController.ts

```diff
@@ -667,144 +614,48 @@ export class TokensController extends BaseController<
     asset: Token,
     type: string,
     interactingAddress?: string,
-  ): Promise<AssetSuggestionResult> {
+  ): Promise<void> {
     const { selectedAddress } = this.config;
 
-    const suggestedAssetMeta: SuggestedAssetMeta & {
-      interactingAddress: string;
```

Comment:
We don't need to specify that `interactingAddress` is not optional, because `selectedAddress` isn't.

Reply:
Hey @bergarces just saw this right now, I think interactingAddress it's different from selectedAddress, Mobile app wise, selectedAddress is the address you have selected on your wallet, but in the app browser wise, the interacted address could be another one connected to a dapp.